assets/src/blocks/Timeline/URLDescriptionHelp.js

```diff
@@ -0,0 +1,22 @@
+const { __ } = wp.i18n;
+
+export const URLDescriptionHelp =
```

Comment:
Like mentioned in my other comment, this will probably be easier to handle if it's a component and not a string. Something we can change later too if we can live with the div-in-a-p-tag warning.

Reply:
@Inwerpsel @lithrel I created an `HTMLSidebarHelp` component and added the help outside the `help` prop, to avoid the warning.